interface/super/edit_list.php

```diff
@@ -1064,6 +1095,46 @@ function mysubmit() {
                     </select>
                 </div>
 
+                <!--Added filter-->
+                <script type="text/javascript">
+                    function lister() {
+                        var urlFull   = new URL($(location).attr('href'));
+                        var list_from = parseInt($("#list-from").val());
+                        var list_to   = parseInt($("#list-to").val());
+                        var list_id_container = $("#list_id").val();
+
+                        if(list_from > list_to || list_from == list_to){
+                            alert(<?php echo xlj("Please enter a enter valid range"); ?>);
+                            return false;
+                        }
+                        if( parseInt(list_from) >= 0 ){
+                            urlFull.searchParams.delete("list_from");
+                            urlFull.searchParams.set("list_from",list_from);
+                        }
+
+                        if( parseInt(list_from) >= 0 ){
+                            urlFull.searchParams.delete("list_to");
+                            urlFull.searchParams.set("list_to",list_to);
+                        }
+                        if( list_id_container.length > 0 ){
+                            urlFull.searchParams.delete("list_id_container");
+                            urlFull.searchParams.set("list_id_container", list_id_container );
+                        }
+                        window.location.replace(urlFull);
+                    }
+                </script>
+                <?php
+                    $urlFrom = ( isset($_GET["list_from"]) && intval($_GET["list_from"]) >= 0 ? intval($_GET["list_from"]) : (isset($_POST["list_from"]) && intval($_POST["list_from"]) >= 0 ? intval($_POST["list_from"]) : 0) );
+                    $urlTo   = ( isset($_GET["list_to"]) && intval($_GET["list_to"])  >= 0 ? intval($_GET["list_to"]) : (isset($_POST["list_to"]) && intval($_POST["list_to"]) >= 0 ? intval($_POST["list_to"]) : 10) );
```

Comment:
Couldn't this be simplified if use $_REQUEST ?

Reply:
You are right.